Support ES6 default exports in webpack/cosmos configs

ovidiubute opened the pull request:
See #242

ovidiubute commented:
@skidding please review

skidding commented on .gitignore:
```diff
@@ -1,3 +1,5 @@
 packages/*/lib
 node_modules
 test/coverage
+.idea/
```
What is this? If it's not project-related best put it in your global gitignore.

skidding commented on packages/react-cosmos-utils/package.json:
```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-cosmos-utils",
-  "version": "2.0.0-beta.1",
+  "version": "2.0.0-beta.2",
```
No need to update versions. Lerna takes care of them automagically when running the publish cmd.

skidding commented on packages/react-cosmos-webpack/package.json:
```diff
@@ -16,7 +16,8 @@
     "loader-utils": "^0.2.16",
-    "react-cosmos": "^2.0.0-beta.2",
+    "react-cosmos": "^2.0.0-beta.3",
+    "react-cosmos-utils": "^2.0.0-beta.2",
```
No need to update dep versions. Lerna takes care of them automagically when running the publish cmd.

ovidiubute commented:
@skidding `.idea` is the project directory for WebStorm. I removed it from `gitignore`, and I also reverted the package versions, didn't realise you were using Lerna. Please review again.
